<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMainWindow</class>
 <widget class="QMainWindow" name="AdminMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Marlene Admin</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="cur_passphrase_edit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::PasswordEchoOnEdit</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Passphrase</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_passphrase_edit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::PasswordEchoOnEdit</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>New Passphrase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_new_passphrase_btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Passphrase</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_pin_edit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>9999</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::PasswordEchoOnEdit</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>New PIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_new_pin_btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set New PIN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="contact_info_edit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Contact Info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_contact_info_btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Contact Info</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>621</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="host_edit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="port_edit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>99999</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hostname</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_host_data_btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>270</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Host Data</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wallet_pass_edit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::PasswordEchoOnEdit</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wallet Passphrase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_wallet_passphrase_btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>320</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Wallet Passphrase</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>621</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="query_card_btn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Query Card</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_pubkey_btn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Public Key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_factory_defaults_btn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Card to Factory Defaults</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>set_new_passphrase_btn</sender>
   <signal>clicked()</signal>
   <receiver>AdminMainWindow</receiver>
   <slot>onSetNewPassphrase()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set_new_pin_btn</sender>
   <signal>clicked()</signal>
   <receiver>AdminMainWindow</receiver>
   <slot>onSetNewPIN()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set_contact_info_btn</sender>
   <signal>clicked()</signal>
   <receiver>AdminMainWindow</receiver>
   <slot>onSetContactInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set_host_data_btn</sender>
   <signal>clicked()</signal>
   <receiver>AdminMainWindow</receiver>
   <slot>onSetHostData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set_wallet_passphrase_btn</sender>
   <signal>clicked()</signal>
   <receiver>AdminMainWindow</receiver>
   <slot>onSetWalletPassphrase()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set_factory_defaults_btn</sender>
   <signal>clicked()</signal>
   <receiver>AdminMainWindow</receiver>
   <slot>onSetFactoryDefaults()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>query_card_btn</sender>
   <signal>clicked()</signal>
   <receiver>AdminMainWindow</receiver>
   <slot>onQueryCard()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set_pubkey_btn</sender>
   <signal>clicked()</signal>
   <receiver>AdminMainWindow</receiver>
   <slot>onSetPublicKey()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSetNewPassphrase()</slot>
  <slot>onSetNewPIN()</slot>
  <slot>onSetContactInfo()</slot>
  <slot>onSetHostData()</slot>
  <slot>onSetWalletPassphrase()</slot>
  <slot>onSetFactoryDefaults()</slot>
  <slot>onSetPublicKey()</slot>
  <slot>onQueryCard()</slot>
 </slots>
</ui>
